lua/please/runners.lua

```diff
@@ -112,4 +112,38 @@ runners.popup = function(cmd, args)
   job:start()
 end
 
+---Runs a command with the given args in tmux.
+---@param cmd string: Command to run.
+---@param args string[]: Args to pass to the command.
+---@param tmux_pre string[]: prefix to the tmux command (e.g. for setting options).
+---@param tmux_args string[]: Args to pass to tmux.
+runners.tmux = function(cmd, args, tmux_pre, tmux_args)
```

Comment:
```suggestion
runners.tmux = function(cmd, args, opts)
```

wdyt about this? i think this should make it easier to swap out `runners.popup` and `runners.tmux` at some point in the future so can call stuff like
```lua
please.build {
  runner = runners.tmux, 
  runner_opts = {
    tmux_args = { 'split-window', '-h' },
  },
}
```